instat/static/InstatObject/R/stand_alone_functions.R

```diff
@@ -2507,4 +2507,17 @@ slopegraph <- function(data, x, y, colour, data_label = NULL,
                     force = 0.5, max.iter = 3000) +
     ggplot2::geom_label(ggplot2::aes_string(label = Ndata_label), size = data_text_size, label.padding = unit(data_label_padding, "lines"),
                label.size = data_label_line_size, colour = data_text_colour, fill = data_label_fill_colour)
-}
\ No newline at end of file
+}
+
+# Returns a three-letter string representing a specific quarter in a year (e.g. "JFM", "AMJ" etc.). 
+get_quarter_label <-   function(quarter, start_month){
+  if (!start_month %in% 1:12) stop(start_month, " is an invalid start month")
+  if (!all(quarter %in% 1:4)) stop("All values must be in range of 1:4")
```

Comment:
Make the messages more consistent and useful?
```suggestion
  if (!start_month %in% 1:12) stop(start_month, " is an invalid start month, must be in range of 1:12")
  if (!all(quarter %in% 1:4)) stop(quarter, " is an invalid quarter, must be in range of 1:4")
```